plugins/search/providers/elastic_search/lib/model/items/ESearchEntryItem.php

```diff
@@ -156,7 +156,7 @@ public function getSingleItemSearchQuery(&$entryQuery, $allowedSearchTypes, &$qu
 		switch ($this->getItemType())
 		{
 			case ESearchItemType::EXACT_MATCH:
-				$subQuery = kESearchQueryManager::getExactMatchQuery($this, $this->getFieldName(), $allowedSearchTypes, $queryAttributes);
+				$subQuery = $this->getExactMatchQuery($this, $this->getFieldName(), $allowedSearchTypes, $queryAttributes);
```

Comment:
better change to $this->getExactMatchQuery($allowedSearchTypes, $queryAttributes);